supersetapiclient/base.py

```diff
@@ -50,7 +51,15 @@ def from_json(cls, json: dict):
             Object: return the related object
         """
         field_names = cls.field_names()
-        return cls(**{k: v for k, v in json.items() if k in field_names})
+        new_object = cls(**{k: v for k, v in json.items() if k in field_names})
+
+        for field_name, clazz in new_object.JSON_AS_OBJECT_FIELDS.items():
```

Comment:
forgot another occurrence
```suggestion
        for field_name, clazz in new_object.LIST_OF_OBJECT_FIELDS.items():
```